agent/proxycfg/state.go

```diff
@@ -551,3 +554,56 @@ func watchMeshGateway(ctx context.Context, opts gatewayWatchOpts) error {
 		EnterpriseMeta: *structs.DefaultEnterpriseMetaInPartition(opts.key.Partition),
 	}, correlationId, opts.notifyCh)
 }
+
+func reconcilePeeringWatches(
+	compiledDiscoveryChains map[UpstreamID]*structs.CompiledDiscoveryChain,
+	upstreams map[UpstreamID]*structs.Upstream,
+	peeredUpstreams map[UpstreamID]struct{},
+	peerUpstreamEndpoints watch.Map[UpstreamID, structs.CheckServiceNodes],
+	upstreamPeerTrustBundles watch.Map[PeerName, *pbpeering.PeeringTrustBundle],
+	logger hclog.Logger) {
+	peeredChainTargets := make(map[UpstreamID]struct{})
```

Comment:
#nit just for improved readability
```suggestion
	logger hclog.Logger) {
	
	peeredChainTargets := make(map[UpstreamID]struct{})
```